task_5/Vanya.Ryabchenko/src/main/java/Task_5.java

```diff
@@ -0,0 +1,25 @@
+public class Task_5 {
```

Comment:
You don't need so many variables to count a sum of them. What if there was a number of 1000 digit?
Try to redo the task using today's lesson approach. Do not just copy the code. Try to work it out by yourself from the beginning